client/src/styles/App.css

```diff
@@ -15,3 +15,19 @@
   color: whitesmoke;
   width: 4.5rem;
 }
+
+.nav-tag-line {
+  color: white;
+  font-style: italic;
+}
+
+.navbar-dark .navbar-brand {
+  color: aqua;
+  font-size: 32px;
+}
```

Comment:
not sure if these are needed anymore, since I changed the styling of the navbar. will revisit this tomorrow.